packages/loot-core/src/platform/server/indexeddb/index.web.ts

```diff
@@ -1,147 +0,0 @@
-// @ts-strict-ignore
-import type * as T from '.';
-
-let openedDb = _openDatabase();
-
-// The web version uses IndexedDB to store data
-function _openDatabase() {
-  return new Promise((resolve, reject) => {
-    const dbVersion = 9;
-    const openRequest = indexedDB.open('actual', dbVersion);
-
-    openRequest.onupgradeneeded = function (e) {
-      // @ts-expect-error EventTarget needs refinement
-      const db: IDBDatabase = e.target.result;
-
-      // Remove old stores
-      if (db.objectStoreNames.contains('filesystem')) {
-        // @ts-expect-error deleteObjectStore does not seem to accept options
-        db.deleteObjectStore('filesystem', { keyPath: 'filepath' });
```

Comment:
This line has changed (2nd argument has been removed as it doesn't actually do anything) - docs https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase/deleteObjectStore